example/screens/ButtonScreen.js

```diff
@@ -1,213 +1,232 @@
-import React, {Component} from 'react';
-
+import React from 'react';
 import {
   View,
-  Text,
   StyleSheet,
   ScrollView,
-  Alert
+  Alert,
 } from 'react-native';
-
-import {RkButton, RkText, RkTheme} from 'react-native-ui-kitten';
-
-import {UtilStyles} from '../style/styles';
+import {
+  RkButton,
+  RkText,
+  RkTheme,
+} from 'react-native-ui-kitten';
 import Icon from 'react-native-vector-icons/FontAwesome';
+import { UtilStyles } from '../style/styles';
 
-export class ButtonScreen extends Component {
+export class ButtonScreen extends React.Component {
   static navigationOptions = {
-    title: 'Buttons'
+    title: 'Buttons',
   };
 
-  _onPress() {
+  onEventButtonPress = () => {
     Alert.alert('onPress');
-  }
+  };
 
-  _onLongPress() {
+  onEventButtonLongPress = () => {
     Alert.alert('onLongPress');
-  }
+  };
 
-  render() {
-    return (
-      <ScrollView
-        automaticallyAdjustContentInsets={true}
-        style={UtilStyles.container}>
-        <View style={[UtilStyles.section, UtilStyles.bordered]}>
-          <RkText rkType='header'>Basic button</RkText>
-          <View style={UtilStyles.rowContainer}>
-            <RkText>Just create button element with text inside.</RkText>
-          </View>
-          <View style={[UtilStyles.rowContainer, {alignItems: 'center'}]}>
-            <RkButton>Button</RkButton>
-            <View style={[UtilStyles.tab]}>
-              <RkText rkType='danger'>{"<RkButton>"}</RkText>
-              <RkText>Button</RkText>
-              <RkText rkType='danger'>{"</RkButton>"}</RkText>
-            </View>
+  render = () => (
```

Comment:
Do we really need to define `render` method as property?
We never call it directly.